src/lib/mina/fetch.unit-test.ts

```diff
@@ -136,96 +137,1181 @@ describe('Fetch', async (t) => {
+            hash: '10375799052052966330950685338047579489788902317193326634260491570085922237258',
+          },
+          {
+            actions: [
+              [
+                '6',
+                '6',
+                '6',
+                '6',
+                '6',
+                '7',
+                '10329935204614394060743187454419407441282038590465076620570686354885776266705',
+                '1',
+                '2',
+              ],
+            ],
+            hash: '28196269303668651416346699879165004850139154523393478734820585068142981031732',
+          },
+          {
+            actions: [
+              [
+                '2',
+                '8',
+                '2',
+                '8',
+                '2',
+                '8',
+                '13935000824120239057894068646248236946420165139009401302919178322058796491040',
+                '1',
+                '1',
+              ],
+            ],
+            hash: '6729748500965556777838399631123462458572881535784863605229371522514436976852',
+          },
+          {
+            actions: [
+              [
+                '1',
+                '2',
+                '5',
+                '2',
+                '2',
+                '5',
+                '10329935204614394060743187454419407441282038590465076620570686354885776266705',
+                '1',
+                '1',
+              ],
+            ],
+            hash: '14719305835147967404874431642690015496919031381380405675285238602024771385112',
+          },
+          {
+            actions: [
+              [
+                '5',
+                '7',
+                '9',
+                '4',
+                '1',
+                '7',
+                '10329935204614394060743187454419407441282038590465076620570686354885776266705',
+                '1',
+                '1',
+              ],
+            ],
+            hash: '23502921406421697988804387941348397070115790652527000864921063117727632274725',
+          },
+          {
+            actions: [
+              [
+                '3',
+                '1',
+                '3',
+                '1',
+                '3',
+                '1',
+                '3358872557945087289287973916777076282977688673157429840747067769412340487925',
+                '1',
+                '1',
+              ],
+            ],
+            hash: '14235722333149067143537863043740008100246151900030018710036040299525674834117',
+          },
+          {
+            actions: [
+              [
+                '9',
+                '8',
+                '5',
+                '3',
+                '2',
+                '7',
+                '5706376253943510402705579272467768662733130021399469094173930489992680696927',
+                '1',
+                '1',
+              ],
+            ],
+            hash: '3432500319723012006583645985859849032413742507994268064744704124629760504866',
+          },
+        ]);
+      });
```

Comment:
this is very long and makes it very hard to read, can we clean this up a little?

Reply:
I don't know what you mean by clean it up?  Do you mean move the payload to a JSON file and read it in?

I would push back on removing this test or trying to prune it to its most useful bits.  Since it is a raw input-output from a real edge case, I think it's worth leaving in full.  It's not really meant to be readable, but it will flag regressions.

Reply:
I don't want to remove it but I would like to make it more readable (dedicated JSON file for the payloads, for example)